src/coreclr/jit/compiler.cpp

```diff
@@ -6709,25 +6705,29 @@ int Compiler::compCompileHelper(CORINFO_MODULE_HANDLE classPtr,
         }
     }
 
-#ifdef DEBUG
-    if ((JitConfig.DumpJittedMethods() == 1) && !compIsForInlining())
+    if (JitConfig.DumpJittedMethods() && !compIsForInlining())
     {
-        enum
-        {
-            BUFSIZE = 20
-        };
-        char osrBuffer[BUFSIZE] = {0};
+#ifdef DEBUG
+        const int BUFSIZE            = 20;
+        char      osrBuffer[BUFSIZE] = {0};
         if (opts.IsOSR())
         {
             // Tiering name already includes "OSR", we just want the IL offset
-            //
             sprintf_s(osrBuffer, BUFSIZE, " @0x%x", info.compILEntry);
         }
 
-        printf("Compiling %4d %s::%s, IL size = %u, hash=0x%08x %s%s%s\n", Compiler::jitTotalMethodCompiled,
-               info.compClassName, info.compMethodName, info.compILCodeSize, info.compMethodHash(),
-               compGetTieringName(), osrBuffer, compGetStressMessage());
+        printf("JIT Compiling %4d [%s]%s, ILsize=%u, hash=0x%08x %s%s\n", Compiler::jitTotalMethodCompiled,
+               compGetTieringName(), info.compFullName, info.compILCodeSize, info.compMethodHash(), osrBuffer,
+               compGetStressMessage());
+#else
+        const char* methodName = info.compCompHnd->getMethodName(info.compMethodHnd, nullptr);
+        const char* className  = info.compCompHnd->getClassName(info.compClassHnd);
+        printf("JIT compiling %4d [%s]%s:%s, ILsize=%u\n", Compiler::jitTotalMethodCompiled, compGetTieringName(),
```

Comment:
If you go by [the docs](https://github.com/dotnet/runtime/blob/main/docs/design/coreclr/jit/viewing-jit-dumps.md#specifying-method-names) (which we should update), it is double colon. Curiously enough though, empirically, both `::` and `:` seem to work...

Reply:
Right, but since both work I removed extra : to match what `compFullName` prints